__tests__/app.test.js

```diff
@@ -165,6 +163,23 @@ describe("GET/api/articles/:article_id", () => {
     });
 })
 
+describe("GET /api/users", () => {
+  test("200: Should respond with all users in an array of objects containing the properties username, name and avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const {users} = body;
+        expect(users.length).toBeGreaterThan(0)
+        users.forEach((user) => {
+          expect(typeof user.username).toBe("string");
+          expect(typeof user.name).toBe("string");
+          expect(typeof user.avatar_url).toBe("string");
+        }); //Do i need to return the users without their id?
```

Comment:
This is good, but I think the use of a different matcher would make it even better. 

https://jestjs.io/docs/expect

The jest docs show a plethora of useful matchers that you might use.